### ISSUE TITLE
All landuse needs sort_key (default)

### ISSUE THREAD

**Issue by nvkelso** [mapzen/vector-datasource#364]
In this tile, the grass don't define sort_key:

http://vector.dev.mapzen.com/osm/all/17/38588/49285.topojson?api_key=vector-tiles-HqUVidw

```
{
type: "Polygon",
properties: {
id: 372703843,
source: "openstreetmap.org",
kind: "grass",
min_zoom: 12.7632,
area: 507
},
arcs: [
[
38
]
]
},
```

**Comment by zerebubuth** [comment]
Landuse sort keys are only defined for kinds in [this list](https://github.com/mapzen/TileStache/blob/integration-1/TileStache/Goodies/VecTiles/transform.py#L1086-L1117), which doesn't include grass. What `sort_key` would you like it to have -- or is there a default that should get applied?

**Comment by nvkelso** [comment]
It should default to `1` (an int), since earth is `0` and landuse should default sort above that.

**Comment by nvkelso** [comment]
With Brett's help I've worked around this in the scene file so this doesn't need to git fixed ASAP.

**Comment by nvkelso** [comment]
I've sketched out proposed sort keys in the attached Open Office document. 

[sort_key.ods.zip](https://github.com/mapzen/vector-datasource/files/148192/sort_key.ods.zip)

This is based on what we're doing now in server code and how that is sometimes hacked in the scene file to achieve the cartographic design. 

**tl;dr**

* Moves the current sort_key range from [0,40] to [0,405].
* All feature kinds now have `sort_key` values, "no" duplicate values (at least for landuse, some for roads)

===

**Design goals:**

* Allow for data "underlays" at various places in the z-orders between logical groupings (mostly between layers).
* Allow future growth in each layer's `kind` values.
* Allow some intermixing of features between layers.
* Live within a z-fighting world, but not go crazy with per tile feature geom area sorts.

@zerebubuth, @rmarianski, @bcamper, @blair1618, @meetar, @sensescape to sanity check

**Comment by nvkelso** [comment]
Related: https://github.com/mapzen/vector-datasource/issues/446.
